layouts/css/_utils.scss

```diff
@@ -14,6 +22,18 @@
   }
 }
 
+.full-width {
+  width: 100%;
+}
+
+.small {
+  font-size: 10px;
```

Comment:
Should this be rem too?

Reply:
Good catch, I'll fix this. This is due to the rebase after the switch to Sass.